Experimental/Anticipation Sample/Assets/DefaultNetworkPrefabs.asset

```diff
@@ -0,0 +1,21 @@
+%YAML 1.1
```

Comment:
Since this sample doesn't spawn players, this can be removed, right? It references the BootstrapPlayer from Netcode for GameObjects package, which seems unused.
At the same time, we could disable the toggle in the Netcode for GameObjects project settings so that this file doesn't get auto-created.

Reply:
done